shared/src/business/useCases/getCasesForUserInteractor.ts

```diff
@@ -90,7 +91,18 @@ const sortAndFilterCases = (nestedCases, caseType: 'open' | 'closed') => {
       } else {
         return compareISODateStrings(b.createdAt, a.createdAt);
       }
-    });
+    })
+    .map(nestedCase => ({
+      ...new UserCase(nestedCase).toRawObject(),
+      consolidatedCases: nestedCase.consolidatedCases
```

Comment:
Previously to solve this problem, we have made DTOs that specify the fields needed for that view and provided them in the request so that `apiGatewayHelper` can use them to filter the data being returned. Could we make use of that strategy here at all?

Reply:
Possibly, but more work would have to be made to either `apiGatewayHelper` or the interactor (or even the whole endpoint) because this interactor currently returns two objects:

```
return { closedCaseList: sortedClosedCases, openCaseList: sortedOpenCases };
```

Reply:
Thanks, that makes sense. The more I thought about it, the more I thought adding this to the UserCase entity makes more sense.

Do any tests need to be added or updated after this change?

Reply:
Good idear, I've included one to cover ensuring `docketEntries` will be undefined.